src/shared/components/annotation/genomeNexus/MutationAssessor.tsx

```diff
@@ -65,6 +67,7 @@ export default class MutationAssessor extends React.Component<IMutationAssessorP
 		const impact = maData.functionalImpact? (
 			<div>
 				<table className={tooltipStyles['ma-tooltip-table']}>
+					<tr><td>Source</td><td><a href="http://mutationassessor.org/r3">MutationAssessor</a></td></tr>
 					<tr><td>Impact</td><td><span className={mutationAssessorColumn[`ma-${maData.functionalImpact}`]}>{maData.functionalImpact}</span></td></tr>
 					{maData.functionalImpactScore? <tr><td>Score</td><td><b>{maData.functionalImpactScore.toFixed(2)}</b></td></tr> : null}
```

Comment:
instead of ternary operator, you should be able to use { (condition) && (<whatever />) }